src/eos-updater-fetch.c

```diff
@@ -82,15 +109,14 @@ content_fetch (GTask *task,
   EosUpdater *updater = EOS_UPDATER (object);
   EosUpdaterData *data = task_data;
   OstreeRepo *repo = data->repo;
-  OstreeRepoPullFlags flags = OSTREE_REPO_PULL_FLAGS_NONE;
   g_autoptr(OstreeAsyncProgress) progress = NULL;
   GError *error = NULL;
   const gchar *refspec;
-  g_autofree gchar *src = NULL;
+  g_autofree gchar *remote = NULL;
   g_autofree gchar *ref = NULL;
   const gchar *sum;
```

Comment:
I assume this variable is meant to refer to a checksum. Would it be better to call it `checksum` in this case? I thought it was was a numerical sum when I was looking over this code initially, so it could be confusing for other maintainers.